<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RocketUpdateDialog</class>
 <widget class="QDialog" name="RocketUpdateDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update Available</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTextEdit" name="updateEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="takeMeButton" >
       <property name="text" >
        <string>Take Me to Update Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remindMeButton" >
       <property name="text" >
        <string>Remind Me Later</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dontTellMeButton" >
       <property name="text" >
        <string>Don't Tell Me Again</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="" />
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>315</y>
     <width>485</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>remindMeButton</sender>
   <signal>clicked()</signal>
   <receiver>RocketUpdateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel" >
     <x>440</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
